cmd/reconstructSeq/reconstructSeq.go

```diff
@@ -10,7 +10,10 @@ import (
 )
 
 func ReconstructSeq(newickInput string, fastaInput string, outputFilename string) {
-	tree := expandedTree.ReadTree(newickInput, fastaInput)
+	tree, err := expandedTree.ReadTree(newickInput, fastaInput)
+	if err != nil {
```

Comment:
You can use the one-line check of exception.FatalOnErr(err).  I am a little surprised that "%e" compiles.  It seems like this would convert your error into a floating point number in scientific notation.  Maybe "%s" is what you want?